ui/admin/tests/acceptance/credential-store/delete-test.js

```diff
@@ -102,9 +98,10 @@ module('Acceptance | credential-stores | delete', function (hooks) {
         (item) => item !== 'delete',
       );
     await visit(urls.vaultCredentialStore);
-    await click(MANAGE_DROPDOWN_SELECTOR);
 
-    assert.dom(DELETE_ACTION_SELECTOR).doesNotExist();
+    await click(selectors.MANAGE_DROPDOWN);
+
+    assert.dom(selectors.DELETE_ACTION).doesNotExist();
```

Comment:
should we switch these and the ones below to isNotVisible?

Reply:
I had considered that but in this case, the items we are looking for actually should not exist at all on the dom. So I felt it made sense to use `doesNotExist()`. If we wanted to temporarily hide something and verify that it is hidden is when we should use `isNotVisible()`. I hope that makes sense and happy to discuss further.

Reply:
yea that makes sense